docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: "3"
+
+services:
+  # Etherpad
+  etherpad:
+    build:
+      context: .
+    container_name: etherpad
+    restart: always
+    environment:
+      - API_KEY=${API_KEY}
+      - SESSION_KEY=${SESSION_KEY}
+      - TITLE=${TITLE}
+      - ADMIN_PASSWORD=${ADMIN_PASSWORD}
+      - DB_TYPE=redis
+      - DB_HOST=redis
+      - DB_NAME=0
+      - DB_PORT=6379
+      - NODE_ENV=production
+      - EDIT_ONLY=false
+      - REQUIRE_SESSION=false
```

Comment:
Setting these to `true` on your standalone etherpad will make the application won't work and it is expected.. Please see `settings.json.docker` Line 229.

With our instances in the wild, the value is set to `true`.